src/util.ts

```diff
@@ -712,6 +713,70 @@ export class Util {
         }
     }
 
+    /**
+     * Small tokenizer for bs:disable comments
+     */
+    public tokenizeBsDisableComment(token: Token) {
```

Comment:
Simple little tokenizer/parser function that makes dealing with disable comments easier.